starter-code/css/main.css

```diff
@@ -6,5 +6,98 @@ light blue: #1c5380
 
 body {
   font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
-  margin: 0;
+  margin: 0 auto;
+  background-image: url("../images/bgtile.png");
+  background-repeat: repeat-x;
```

Comment:
Don't directly put a background-image on your body, that's not very clean to write it like this :) 
Prefer creating a big section which contains all of the body's content.
Then add CSS properties to center everything on the page : 
```
.big-new-section{
  height: 100vh;
  display: flex;
  align-items: center;
  justify-content: center;
  flex-direction: column;
}